src/Sulu/Bundle/AdminBundle/Resources/js/containers/Login/ResetPasswordForm.js

```diff
@@ -34,6 +42,10 @@ class ResetPasswordForm extends React.Component<Props> {
         return !(this.password1 && this.password2);
     }
 
+    @computed get matchPattern(): boolean {
```

Comment:
think i would not add a property for this but just call the `userStore.validatePassword` directly. at the moment, the validation will be executed on every keystroke